OMEdit/OMEditGUI/OMEditGUI.win.config.pri

```diff
@@ -48,8 +48,10 @@ CONFIG(release, debug|release) { # release
   # win32 vs. win64
   contains(QT_ARCH, i386) { # 32-bit
     LIBS += -L$$(OMDEV)/tools/msys/mingw32/lib/binutils -L$$(OMDEV)/tools/msys/mingw32/bin
+    QMAKE_CXXFLAGS += -I$$(OMDEV)/tools/msys/mingw32/include/binutils
   } else { # 64-bit
     LIBS += -L$$(OMDEV)/tools/msys/mingw64/lib/binutils -L$$(OMDEV)/tools/msys/mingw64/bin
+    QMAKE_CXXFLAGS += -I$$(OMDEV)/tools/msys/mingw64/include/binutils
```

Comment:
We should do this for OMEditLib as well. Probably here 

https://github.com/OpenModelica/OpenModelica/blob/2c2d71103bb0c7c67417ff21a18581fb4b820ae8/OMEdit/OMEditLIB/OMEditLIB.pro#L74

but I am not sure.

Reply:
Also we need to use `QMAKE_CFLAGS` instead of  `QMAKE_CXXFLAGS`.

Reply:
Is there no `QMAKE_CPPFLAGS`?

Reply:
At least none is used.